src/constants.ts

```diff
@@ -0,0 +1,25 @@
+const daysNumberOnTimeGrid: number = 7;
```

Comment:
You don't need to specify types for the constants unless you want to implement certain interface. There is a thing called type inference where TypeScript language will find the type out of the box.

```suggestion
const daysNumberOnTimeGrid = 7;
```

<img width="821" alt="Screenshot 2024-08-08 at 10 59 27" src="https://github.com/user-attachments/assets/cbb97b20-8193-4f69-b4dd-bc766f91f1d8">